app/controllers/admin/loans_controller.rb

```diff
@@ -117,7 +117,8 @@ def print
     authorize @loan, :show?
     @print_view = true
     @mode = params[:mode]
-    @first_image = @loan.media.find {|item| item.kind_value == 'image'}
+    @images = @loan.media.where(kind_value: "image")
+    @image_list = @images.drop(1)
```

Comment:
Cool! I never knew about `drop`...

Reply:
One other thing: if you do `@images.drop(1).each_slice(8).to_a` you'll get an array of arrays of 8 images each. Then you can use that to clean up the view. See below.